enterprise_access/apps/api/v1/views/admin_portal_learner_profile.py

```diff
@@ -0,0 +1,71 @@
+"""
+REST API views for the admin_portal_learner_profile app.
+"""
+import logging
+
+from drf_spectacular.utils import extend_schema
+from rest_framework import permissions, status
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from enterprise_access.apps.admin_portal_learner_profile import api as admin_portal_learner_profile_api
+from enterprise_access.apps.admin_portal_learner_profile import serializers
+
+logger = logging.getLogger(__name__)
+
+
+class AdminLearnerProfileViewSet(ViewSet):
+    """
+    A class that allows admins to look up subscriptions, enrollments, and group
+    memberships for a learner.
+
+    GET /api/v1/admin-view/learner_profile
+
+    Expected params:
+    - user_email (string): The email address for a learner within an enterprise.
+    - lms_user_id (string): The UUID of the LMS user.
```

Comment:
nit: The lms user id, while unique, is not a UUID, which is a specific 128-bit value
```suggestion
    - lms_user_id (string): The unique id of the LMS user.
```